pkg/kdump/kdump-view.jsx

```diff
@@ -269,18 +282,18 @@ export class KdumpPage extends React.Component {
         const target = this.props.kdumpStatus.target;
         let verifyMessage;
         if (!target.multipleTargets) {
-            let path = target.path || "/var/crash";
+            const path = getLocation(target);
+
             if (target.type === "local") {
                 verifyMessage = fmt_to_fragments(
                     ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
                     <span className="pf-v5-u-font-family-monospace-vf">{path}</span>,
                     <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
             } else if (target.type === "ssh" || target.type == "nfs") {
-                path = (target.type == "nfs" && path[0] !== '/') ? '/' + path : path;
                 verifyMessage = fmt_to_fragments(
                     ' ' + _("Results of the crash will be copied through $0 to $1 as $2, if kdump is properly configured."),
                     <span className="pf-v5-u-font-family-monospace-vf">{target.type === "ssh" ? "SSH" : "NFS"}</span>,
-                    <span className="pf-v5-u-font-family-monospace-vf">{`${target.server}:${target.type === "nfs" ? target.export + path : path}`}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">{path}</span>,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19240-20230824-135223-f118a980-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)